.github/scripts/test-openjdk-pullrequests.py

```diff
@@ -801,7 +801,31 @@ def get_pr_to_test(untested_prs, visited):
         # Search runs for non-expired artifact
         for run in runs["workflow_runs"]:
             run_id = run["id"]
+            artifacts_obj_static = gh_api(["--paginate", f"/repos/{repo}/actions/runs/{run_id}/artifacts?name={_artifact_to_test}-static"])
```

Comment:
Looks good. You could consider adding a `download_and_extract_bundle` helper although maybe there's not enough shared logic between the JDK main bundle and static-libs bundle to warrant this.

Reply:
not enough shared logic -- static libs are bundled directly in zip, jdk are bundled in a tgz then in a zip

Reply:
I was wrong. It was due to mac automatically uncompresses the sole tgz within the zip. We might consider extracting the logic now that there is a bit more shared logic